src/Pages/Layout.tsx

```diff
@@ -24,8 +25,13 @@ export default function Layout() {
     const readNotifications = useSelector<RootState, number>(s => s.login.readNotifications);
     const dms = useSelector<RootState, RawEvent[]>(s => s.login.dms);
     const prefs = useSelector<RootState, UserPreferences>(s => s.login.preferences);
+    const pub = useEventPublisher();
     useLoginFeed();
 
+    useMemo(() => {
```

Comment:
we are not using result so can use `useEffect`

```suggestion
    useEffect(() => {
```